clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -79,7 +79,7 @@ class IoUtil(storage: Storage) {
   def writeGoogleObjectData(data: String, location: URI): Unit = {
     getBlob(location) match {
       case Some(blob) =>
-        for (channel <- resource.managed(blob.writer())) {
+        blob.writer().autoClosed { channel =>
```

Comment:
I found a better-files equivalent, so I dropped scala-arm

Reply:
I looked in better files for this but didn't see it, makes sense they would have it.